google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -127,6 +129,18 @@ public final String getContent() {
     return content;
   }
 
+  /**
+   * Returns the HTTP response or {@code null} for none as an input stream.
+   *
+   * @since 1.28
+   */
+  public final InputStream getContentAsInputStream() {
+    if (content == null) {
+      return null;
+    }
+    return new ByteArrayInputStream(content.getBytes());
```

Comment:
This doesn't seem right.  Please confirm that this works with an image, or some other binary values.

Reply:
The change uses ByteArrayInputStream  because the Builder that builds the content uses the HttpResponse method parseAsString 
https://github.com/googleapis/google-http-java-client/blob/380a60b499b845a4ba423e1105a469dabfdf4ad2/google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java#L175

https://github.com/googleapis/google-http-java-client/blob/a7ee06950b8309787127340e90be0b528c197378/google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java#L514
which uses ByteArrayOutputStream 

The change just reverts the string back to the input stream. Please correct me if I'm wrong.

Reply:
@andrey-qlogic This is not exactly correct it will not convert back to bytes correctly if charset is not same and error prone too. Can you please make change to store original content in to byte and then when somebody calls getContent convert to string and when asking for stream pass it as stream?

Reply:
Done